lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+module Players
+
+  class Human < Player
+
+    def move(board)
+      puts puts "Please select from 1-9"
+      input = gets.strip
+    end
```

Comment:
Here, we could just write `gets.strip` as the last line.  No need for the `input` variable.